plugins/inputs/http_listener_v2/http_listener_v2.go

```diff
@@ -50,12 +51,15 @@ type HTTPListenerV2 struct {
 	BasicUsername  string            `toml:"basic_username"`
 	BasicPassword  string            `toml:"basic_password"`
 	HTTPHeaderTags map[string]string `toml:"http_header_tags"`
+
 	tlsint.ServerConfig
+	tlsConf *tls.Config
```

Comment:
What is the purpose of saving the tls config? It doesn't seem to be used anywhere else besides during the setup of the listener.

Reply:
As far as I can see it, it is being used in `createHTTPServer` which is called in `Start` to set the tls config in the http server itself and since there's not way to pass arguments between `Init` and `Start` it has to be either done as is or we'd have to call 

```go
tlsConf, err := h.ServerConfig.TLSConfig()
```

twice. WDYT @sspaink ?

Reply:
Your right, what you have now makes the most sense.